CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
+## Forest v0.5.0 (unreleased, scheduled for 2022-12-30)
 
-## Forest v0.3.0 (unreleased)
+## Forest v0.4.0 (2022-09-30)
```

Comment:
Should we also update crate version of `forest`? Currently `forest --version` prints `0.2.2+git.xxxx`

Reply:
Indeed, all the crates need to have their version bumped.